pages/adult-form.js

```diff
@@ -0,0 +1,20 @@
+import Head from 'next/head';
+import { useRouter } from 'next/router';
+
+const Adult = () => {
+  const router = useRouter();
+  return (
+    <div>
+      <Head>
+        <title>Adult</title>
+        <link rel="icon" href="/favicon.ico" />
+      </Head>
+      <button onClick={() => router.back()}>Back</button>
+      <div className="hero">
```

Comment:
These `classNames` are superfluous as they were defined in the `create-next-app`, we can get rid of them in the next iteration :)